app/models/admin/site.rb

```diff
@@ -0,0 +1,50 @@
+require 'list_processor'
```

Comment:
Move `lib/list_processor.rb` to `app/lib/list_processor.rb` and drop this `require`